### PR TITLE
Fix deletion

### PR DESCRIPTION
**Pull request by coreyfloyd**
Needed to increase the threshold for detecting a deletion pan in contrast to a scrolling pan
Also fading out the deleted item as it moves off screen so it is more obvious what is happening

### CODE REVIEW

File: Wikipedia/UI-V5/WMFEditingCollectionViewLayout.m [hunk 1]
```diff
@@ -371,6 +371,11 @@ - (IBAction)panWithGesture:(UIPanGestureRecognizer*)pan {
                 CGPoint currentCenter = self.movingCellCenter;
                 currentCenter.x               += translation.x;
                 self.movingSnapshotView.center = currentCenter;
+                CGFloat alpha = CGRectGetMaxX(self.movingSnapshotView.frame) / self.collectionView.bounds.size.width;
+                if(alpha > 1){
+                    alpha = 1 - (alpha - 1);
```
Comment (bgerstle) [inline Wikipedia/UI-V5/WMFEditingCollectionViewLayout.m]: can you add some comments explaining what is going on here? it looks like you're "capping" the alpha at 1, but it's not clear why you're subtracting the "remainder" above 1 from 1.

**Comment by bgerstle** [comment]
would like some comments on the alpha calculation, but otherwise :+1:

**Comment by coreyfloyd** [comment]
@bgerstle this is working well for me with this new value.

**Comment by bgerstle** [comment]
@coreyfloyd :+1:

**Comment by coreyfloyd** [comment]
@bgerstle updated to a the more performant implementation for the indexPathForTitle. 

Obviously we need to dry all this up later.

**Pull request merged**